examples/common/python/utility/file_utils.py

```diff
@@ -55,7 +60,32 @@ def read_json_file(input_file, data_dir = ['./', '../', '/']) :
     """
     Utility function to read a JSON file
     """
-    file_name = find_file_in_path(input_file, data_dir)
+    file_name = find_file_in_paths(input_file, data_dir)
     with open(file_name, "r") as input_json_file :
-        input_json = input_json_file.read().rstrip('\n')
-    return input_json
+        input_lines = input_json_file.read().splitlines()
+    flattened_json = ""
+    for line in input_lines :
+        flattened_json += line
+    return flattened_json
+
+#---------------------------------------------------------------------------------------------
+def write_result_data_to_json_file(file_name, input_data, data_dir ='./') :
+    """
+    Function to store result data as json file
+    Parameters:
+        - file_name is the name in which the file should be stored
+        - input_data is a JSON data which needs to be stored in a file( should have attribute 'result')
+        - data_dir is the directory path to store the file
+    """
+    result_info = dict()
```

Comment:
I think, the preferable way to declare an empty dictionary in python is result_info = {}. This way is faster.
Your way of declaring empty dictionary is good as it provides flexibility in syntax and initialization. But  as you are declaring an empty dict, I recommend to use {}.

Reply:
This again is a moved piece of code Pankaj. However I would rather prefer it to be this way(more verbose the better). Performance might not be an issue here at all as this is a standalone statement(not within a loop as such) and called rather infrequently. By the way I did evaluate the efficiency of the two as well -
```
C:\Users\rranjan3>python -m timeit -n 1000000 -r 5 -v 'dict()'
raw times: 12.4 msec, 8.67 msec, 9.71 msec, 8.77 msec, 8.42 msec

1000000 loops, best of 5: 8.42 nsec per loop

C:\Users\rranjan3>python -m timeit -n 1000000 -r 5 -v '{}'
raw times: 16.2 msec, 8.5 msec, 9.92 msec, 9.88 msec, 9.51 msec

1000000 loops, best of 5: 8.5 nsec per loop
```

They seem to be very much at par.

Reply:
makes sense and good exploration!